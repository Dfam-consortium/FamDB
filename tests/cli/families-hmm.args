families
--format
hmm
-ad
5
