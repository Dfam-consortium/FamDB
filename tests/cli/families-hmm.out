#   <copyright header>
<model1>
MS    TaxId:1 TaxName:root
CC    RepeatMasker Annotations:
CC         Type: 
CC         SubType: 
CC         Species: root
CC         SearchStages: 
CC         BufferStages: 
<model2>
MS    TaxId:2 TaxName:Order
MS    TaxId:3 TaxName:Other_Order
CC    RepeatMasker Annotations:
CC         Type: 
CC         SubType: 
CC         Species: Order, Other_Order
CC         SearchStages: 
CC         BufferStages: 
<model3>
MS    TaxId:3 TaxName:Other_Order
CC    RepeatMasker Annotations:
CC         Type: 
CC         SubType: 
CC         Species: Other_Order
CC         SearchStages: 30,40
CC         BufferStages: 
